src/components/ComparisonMap/ComparisonMap.vue

```diff
@@ -100,45 +114,32 @@
 
     data () {
       return {
-        dateIndex: -1,
-        oldDateIndex: -1,
+        isoDate: (new Date()).toISOString().substring(0, 10),
```

Comment:
Can you explain this?

Reply:
The Vuetify date picker [accepts ISO 8601 date strings (YYYY-MM-DD)](https://v2.vuetifyjs.com/en/components/date-pickers/#caveats). It was quite unexpected for me. I will see what is the best way to add a comment for that.